integrasjon/rest-klient/src/main/java/no/nav/vedtak/felles/integrasjon/rest/FpApplication.java

```diff
@@ -22,6 +22,7 @@ public enum FpApplication {
     FPWSPROXY,
     FPOVERSIKT,
     FPTILGANG,
+    FTINNTEKTSMELDING,
```

Comment:
Dere kan også bruke en vanlig RestClientConfig med config for uri og scopes fram til appnavn og port er stabil. 
Se KabalKlient for eksempel på kall til app i GCP

Reply:
Ok, avventer å legge det her til appnavn er stabilt. Det kan endres. Lukker denne PRen